catalog.html

```diff
@@ -59,15 +59,15 @@ <h2>фильтр</h2>
         </legend>
         <ul>
            <li>
-             <input type="radio" id="adaptive-grid" name="adaptive" checked>
+             <input type="radio" id="adaptive-grid" name="setka" value="adaptive" checked>
```

Comment:
Эмм, только транслитом не надо писать )
Используй для сетки в этом значении слово `layout`

```suggestion
             <input type="radio" id="adaptive-grid" name="layout" value="adaptive" checked>
```